docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md

```diff
@@ -19,9 +19,6 @@ The Sophos Central Source ingests:
 :::note
 To link the endpoint data to the alert, you can map the `alert.ManagedAgent.ID` field from the alert response with the `endpointID` field from the endpoint response.
 :::
-:::note
-This Source is not available in the [Fed deployment](/docs/api/troubleshooting#Deployments-and-Sumo-Logic-Endpoints).
```

Comment:
Do not delete this note, instead change it as `This Source is available in the [Fed deployment](/docs/api/troubleshooting#Deployments-and-Sumo-Logic-Endpoints).`

Reply:
Updated.